Generator/Templates/GObject/class.sbntxt

```diff
@@ -27,7 +21,7 @@ namespace {{ namespace }}
 {
     // AUTOGENERATED FILE - DO NOT MODIFY
     
-    public{{get_abstract}}partial class {{ name }} {{ (': ' + parent) | get_if_parent }}
+    public partial class {{ name }} {{ (': ' + parent) | get_if_parent }}
```

Comment:
What's the reasoning behind removing abstract classes? Not that I'm against it, but I'm trying to remember what we used them for.

Reply:
I think we did not use it at all. We inserted it "just because". The problem is that we can receive objects of abstract classes from GTK.

In this PR there is the Gst.Parse class which returns an `Element`. In the gir file this class is marked as abstract:

```XML
<class name="Element" c:type="GstElement" parent="Object" abstract="1"
...
```

To be able to create an instance of this object our class can't be abstract.

The global function `parse_launch` returns this `<type name="Element" c:type="GstElement*"/>`. So our best bet is to create an `Element` even if it is abstract.